packages/vaadin-lumo-styles/utilities/spacing.js

```diff
@@ -7,8 +7,20 @@ import { css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 
 export const spacing = css`
   /* === Margin === */
-  .m-auto {
-    margin: auto;
```

Comment:
Why are we removing these auto-margin classes?

Reply:
I just moved it down a bit.